pkg/virt-launcher/virtwrap/efi/efi_test.go

```diff
@@ -0,0 +1,75 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2017 Red Hat, Inc.
+ *
+ */
+
+package efi
+
+import (
+	"os"
+	"path"
+	"path/filepath"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("EFI environment detection", func() {
+	createEFIRoms := func(efiRoms ...string) string {
+		ovmfPath, err := os.MkdirTemp("", "kubevirt-ovmf")
+		Expect(err).To(BeNil())
+
+		for i := range efiRoms {
+			if efiRoms[i] != "" {
+				_, err := os.Create(path.Join(ovmfPath, efiRoms[i]))
```

Comment:
Nit: It seems that `os.Create` also opens the file for writing, shouldn't we close it?

Reply:
Yup indeed